### DISCUSSION TITLE
How to deploy Docker images with OpenID configured?

### DISCUSSION THREAD

**Discussion by dsaffie** [DependencyTrack/dependency-track#1466] [Q&A]
I started with the [OpenID documentation](https://docs.dependencytrack.org/getting-started/openidconnect-configuration/) provided by Dependency-Tracker. Seeing that the Server needs to be deployed separately from the Front-End, I went to both the [Configuration](https://docs.dependencytrack.org/getting-started/configuration/) and [Deploying Docker](https://docs.dependencytrack.org/getting-started/deploy-docker/) pages. However, I'm not seeing instructions for deploying the two Docker images separately. 

I did uncomment and fill out the OpenID section of the docker-compose.yml, then ran `docker-compose up -d`, but that alone (as I somewhat expected) didn't configure OpenID correctly, as when I hit the UI, there as no option to login with OpenID. Just in case, I did clear the browser cache, but no luck.

What I still can't understand is how would I pull both Docker images, make the modifications necessary to the Front-End configuration to use OpenID, and then start them together?

**Comment by AbdelHajou** [comment]
Could you show us the docker-compose section where your OpenID is configured? And perhaps take a look at the server logs to see if there is anything wrong with the OpenID configuration?
